src/shared/components/ExternalLink/ExternalLink.tsx

```diff
@@ -19,9 +20,26 @@ export const ExternalLink = ({
   children,
   design = 'Default',
   iconStyle,
+  type = 'link',
 }: LinkProps) => {
   const { t } = useTranslation();
 
+  if (type === 'button') {
+    return (
+      <Button
+        icon="inspect"
+        iconEnd
+        style={spacing.sapUiTinyMarginBeginEnd}
```

Comment:
in my opinion there is too much space between `Create` and `Learn more`
previously: 
![image](https://github.com/kyma-project/busola/assets/46759954/1c3a870a-d83e-4674-9eac-032523b922a5)

now: 
![image](https://github.com/kyma-project/busola/assets/46759954/fc0a4e47-78d0-4236-9799-608168a19fbb)